packages/block-library/src/query-title/edit.js

```diff
@@ -48,6 +48,33 @@ export default function QueryTitleEdit( {
 			<TagName { ...blockProps }>{ __( 'Archive title' ) }</TagName>
 		);
 	}
+
+	if ( type === 'search' ) {
+		titleElement = (
+			<>
+				<InspectorControls>
+					<PanelBody title={ __( 'Settings' ) }>
+						<ToggleControl
+							label={ __( 'Show search term in title' ) }
+							onChange={ () =>
+								setAttributes( {
+									showSearchTerm: ! showSearchTerm,
+								} )
+							}
+							checked={ showSearchTerm }
+						/>
+					</PanelBody>
+				</InspectorControls>
+
+				<TagName { ...blockProps }>
+					{ showSearchTerm
+						? __( 'Search results for: "search term"' )
```

Comment:
```suggestion
						? __( 'Search results for: %search_term%' )
```

Not sure about my suggestion either but the original is misleading that there is nothing to suggest the text in quotes will be replaced by search term.

Reply:
Yes good point.  Is % used elsewhere in WP to indicate that something is a placeholder?

Reply:
The double quotes are visible on the front too, so they should be kept or removed on the front.

Reply:
```suggestion
						? __( 'Search results for: "search term placeholder"' )
```